app/controllers/concerns/high_voltage/static_page.rb

```diff
@@ -22,14 +22,18 @@ module HighVoltage::StaticPage
     if respond_to?(:caches_page)
       caches_page :show, if: -> { HighVoltage.page_caching }
     end
-
-    hide_action :current_page, :page_finder, :page_finder_factory
```

Comment:
You're right. I misread and missed that `current_page` `page_finder` and `page_finder_factory` are now private so they wouldn't need to use `hide_action` anyways.